demos/python_demos/smart_library_demo/smart_library_demo.py

```diff
@@ -0,0 +1,262 @@
+import sys, os
+import argparse
+import numpy as np
+import cv2 as cv
+import json
+import logging as log
+
+log.basicConfig(stream=sys.stdout, level=log.DEBUG)
+
+sys.path.insert(0,'src/modules')
+import face_recognizer as fr 
+import book_recognizer as br
+
+sys.path.insert(0, 'src/infrastructure')
+from DynamicDatabase import *
+
+uknownID = -1
+count = 0
+
+def CreateArgparse():
```

Comment:
Please, use snake case for function names.

Reply:
Done in commit https://github.com/opencv/open_model_zoo/pull/882/commits/98aad7ead4a363071c097ffa225f9267cac64c5d